test/tests/dom/getCurrentScriptUID.js

```diff
@@ -0,0 +1,60 @@
+/* @flow */
+
+import { getCurrentScriptUID, getCurrentScript, memoize, ATTRIBUTES } from '../../../src';
+
+
+beforeEach(() => {
+    const script = getCurrentScript();
+    script.setAttribute(`${ ATTRIBUTES.NONE }`, '654321');
+});
+
+afterEach(() => {
+    const script = getCurrentScript();
+    script.removeAttribute(`${ ATTRIBUTES.UID }`);
+    script.removeAttribute(`${ ATTRIBUTES.UID }-auto`);
+    memoize.clear();
+});
+
+describe('get current script UID', () => {
+
+    it('should create a data-uid-auto attribute', () => {
+        
+        getCurrentScriptUID();
+
+        const currentScript = getCurrentScript();
+        const uidAttr = currentScript.getAttribute(`${ ATTRIBUTES.UID }-auto`);
+
+        if (!uidAttr) {
+            throw new Error(
+                `Expected result have a 'data-uid-auto' attribute, got none`
```

Comment:
Just a suggestion, maybe:

`should have returned 'data-uid-auto'`, got undefined